.github/workflows/release.yml

```diff
@@ -93,6 +98,23 @@ jobs:
           asset_name: ${{ runner.os }}.tar.gz
           asset_content_type: application/gzip
 
+      - name: Package Checksum
+        shell: bash
+        run: |
+          sha256sum ${{ runner.os }}.tar.gz | awk '{print $1}' > ${{ runner.os }}.sha
```

Comment:
This part can break in Windows (idk if these commands are available in the windows environment), I don't have a Windows machine to test that.

Reply:
Yeah, I don't think we can use `sha256sum` on Windows. We should do [like the compiler does](https://github.com/purescript/purescript/blob/18e3b14c556c5bd58e391531e4f4e511f834c3ff/bundle/build.sh#L48-L57), and use `openssl` on Windows. Quoting from the link above:
```bash
# Calculate the SHA hash
if [ "$OS" = "win64" ]
then
  # msys/mingw does not include shasum. :(
  SHASUM="openssl dgst -sha1"
else
  SHASUM="shasum"
fi

$SHASUM bundle/${OS}.tar.gz > bundle/${OS}.sha
```